templatetags/htk_tags.py

```diff
@@ -302,6 +302,23 @@ def get_request_duration():
     return duration
 
 
+@register.simple_tag()
+def localize(key):
```

Comment:
Please implement the TODO. You can change the the signature to `locale='en-US'` so that it is the default if not passed in.